### PR TITLE
feat: remove command 'redeem' from subscription-manager

### PR DESCRIPTION
**Pull request by grunwmar**
Code lines referencing or using	`redeem` command object were deleted. Associated files defining command `redeem` and test for it were removed.
Command	`redeem` was removed from bash auto completion and man pages.

**Steps to reproduce**
- Clone the repository and switch to the branch `feat/cct-242`
- Install `subscription-manager` according `INSTALL.md`
- Run according to `INSTALL.md` (no `redeem` command appears)
- run `man subscription-manager` (no `redeem` command appears)

CARD ID: CCT-242

### CODE REVIEW

File: man/subscription-manager.8 [hunk 1]
```diff
@@ -92,49 +92,46 @@ must be passed as system arguments in a non-interactive session.
 7. import
 
 .IP
-8. redeem
```
Comment (jirihnidek) [inline man/subscription-manager.8:95]: Please, just remove redeem command from the list. There will be lot of removals of commands. Lot of conflicts. We will fix numbers in final edit of man page.

**Review by jirihnidek — CHANGES_REQUESTED** [review]
Thanks for the PR. It looks good. I have only one small request.

**Review by m-horky — CHANGES_REQUESTED** [review]
Could you revert the numbering for items in the man page, as Jirka suggested? We'll do it in one go later.

Could you also please reference the ticket ID in commit message? It is useful to include this kind of information.

**Review by m-horky — APPROVED** [review]
LGTM

**Review by ptoscano — CHANGES_REQUESTED** [review]
This is generally good; there are few more references to "redeem" to remove from the man page:
- the "REDEEM OPTIONS" section, which can be removed altogether
- in the "DEPRECATED COMMANDS" section, the bit about "activate" that points to "redeem"; "activate" was removed long ago, and leaving the pointer to "redeem" would give a "broken" reference
- the "REDEEMING EXISTING SUBSCRIPTIONS" section, which can be removed altogether

Thanks!

**Comment by ptoscano** [comment]
> in the "DEPRECATED COMMANDS" section, the bit about "activate" that points to "redeem"; "activate" was removed long ago, and leaving the pointer to "redeem" would give a "broken" reference

I just fixed that with #3429; the other bits need to be removed.

Also, the reference to the card must be fixed: it is CCT-424, not CCT-242.

**Comment by ptoscano** [comment]
Obsoleted by #3446, which was based on this.

**Pull request closed without merging**